egs2/TEMPLATE/enh1/README.md

```diff
@@ -1,28 +1,44 @@
 # Speech Enhancement Frontend Recipe
 
-This is the common recipe for for ESPnet2 speech enhancement frontend. Currently, two speech separation recipes (`wsj0_2mix` and `wsj0_2mix_spatialized`) are supported:
+This is the common recipe for for ESPnet2 speech enhancement frontend. Currently, ten speech enhancement/separation recipes are supported:
 ```
 egs2/
+├── chime4
+│   └── enh1
+├── dirha_wsj
+│   └── enh1
+├── libri_css
+│   └── enh1
```

Comment:
Does not exist?

Reply:
This recipe is not yet included in this PR. We will make new commits later.